docs/admin/ProjectManagementDocumentation.md

```diff
@@ -0,0 +1,15 @@
+# Project Management Documentation
+*A document to describe the overview of how we will manage the project*
+
+We will:
```

Comment:
Could these bullets be split under a few subheadings? Code Management, Work Distrebution, Management Techniques, etc.